Makefile

```diff
@@ -38,14 +38,17 @@ LDFLAGS = -L$(EX_FAISS_CACHE_LIB_DIR) -lfaiss
 ifeq ($(shell uname -s), Darwin)
 	LDFLAGS += -flat_namespace -undefined suppress
 	POST_INSTALL = install_name_tool $(EX_FAISS_CACHE_SO) -change @rpath/libfaiss.dylib @loader_path/lib/libfaiss.dylib
+
+	LLVM_PREFIX=$(shell brew --prefix llvm)
```

Comment:
I don't think we should assume folks are using brew. Is there a more programmatic way we can find this?

Reply:
Brew doesn't install to the /user/lib, where CMAKE looks for clang/clang++ to avoid trampling over the defaults. If you want to use brew llvm you need to do it this way.